src/style.css

```diff
@@ -1 +1,114 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(./fonts/Avenir-Book.ttf)
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 600;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+:root {
+--grey: #3d4e61;
+--light-grey: #e1e7ed;
+--white: #fff;
+}
+
+html {
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  font-weight: 300;
+}
+
+.search-form {
+  margin-top: 20px;
+}
+
+.search-form__label {
+  position: relative;
+}
+
+.search-form__label--big {
+  width: 100%;
+  height: 70px;
+}
+
+.search-form__label--small {
+  height: 42px;
+}
+
+.search-form__input {
```

Comment:
should also have bold (600 in your case) font-weight - use `Avenir-Heavy.ttf` for typed text

but since everything else LGTM - approved (no need to re-request review) - this comment only FYI

Reply:
Im not quite understand, how to use Avenir-Heavy.ttf. Can you explain it more please?